docs/guide/nts.md

```diff
@@ -0,0 +1,109 @@
+
+## Configuring an NTS server as time source
+
+Configuring a public NTS server, for example those provided by [netnod](https://www.netnod.se/nts/network-time-security), is fairly straightforward:
+```toml
+[[peer]]
+mode = "nts"
+address = "nts.netnod.se"
+```
+
+The above assumes that the certificate used by the server is accepted through the root certificates of the system. If this is not the case, the root certificate used in the certificate chain of the server needs to be separately provided to ntpd-rs. This can be done through the `certificate-authority` option:
+```toml
+[[peer]]
+mode = "nts"
+address = "my.private.server"
+certificate-authority = "/path/to/server/root/certificate.pem"
+```
+The root certificate file provided in the above example configuration should be a pem encoded list of root certificates that should be accepted for the server in addition to those in the system certificate store.
+
+Currently, ntpd-rs does not support using self-signed certificates. If non public certificates are needed, the best option is to create a private CA locally and use that to sign the server's certificate. Instructions for that are included below in the instructions for configuring ntpd-rs as an NTS server.
+
+## Configuring ntpd-rs as an NTS server
+
+To configure ntpd-rs as a server, we need to configure three things. First, ntpd-rs needs to be configured to work as a normal NTP server. Second, we need to enable the server components that allow key exchange with clients. Finally, we need to configure storage for internal keys to ensure clients don't need to reconnect on a reboot of the server.
+
+The process of setting up an NTS server is significantly harder than configuring an NTS time source. We assume here some experience with [setting up a non-NTS NTP server](TODO), and will assume you already have that running on the target machine.
+
+Furthermore, NTPD-rs currently does not support running an NTS server on a server without a domain name through which its clients reach it. This can however be a local domain name, such as one configured through `/etc/hosts`.
+
+### Certificates
+
+Before setting up the key exchange part of the server, we will first need to create the required key material. We will focus here on locally signing all the certificates for our server. For setting up a public server with NTS support you will need to request a publicly signed certificate. We suggest to use [letsencrypt](https://letsencrypt.org/) for that use case.
```

Comment:
the concept of a "key exchange" has not been introduced at this point. In effect you assume some knowledge of how NTS works under the hood. I think it makes sense to explicitly sketch out how NTS works and introduce some of the core concepts

Reply:
Fixed with larger introduction